IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignment/Documentation.md

```diff
@@ -1,25 +1,78 @@
+
+A single alignment may have: 
+
+* an accompanying vertical alignment, defined along the horizontal alignment in the distance along / z coordinate space  
+* a relative alignment, defined as a span within another alignment and/or at constant or variable offsets  
+* a 3D alignment, either computed from the horizontal and vertical alignment, or extracted from geospatial data.  
   
 Alignments may be aggregated into referents (_IfcReferent_) or derivative alignments. Derivative alignments may be used to indicate dependent alignments, such as an alignment for a bridge that is relative to a parent alignment for a road, where the child _IfcAlignment_ may have _Axis_ set to _IfcOffsetCurveByDistances_ that starts and ends at a span within the extent of the original _IfcAlignment_ defined at the _Axis_ of the parent _IfcAlignment_.  
   
 Alignments may be assigned to groups using _IfcRelAssignsToGroup_, where _IfcGroup_ or subtypes may capture information common to multiple alignments.  
   
 Supported representations of <span class="self-ref">IfcAlignment</span>.Axis are:  
   
-\* _IfcGradientCurve_ as a 3D horizontal and vertical alignment (represented by their alignment segments)  
-\* _IfcCompositeCurve_ as a 2D horizontal alignment (represented by its horizontal alignment segments) without a vertical alignment  
-\* _IfcOffsetCurveByDistances_ as a 2D or 3D curve defined relative to an _IfcAlignmentCurve_ or another _IfcOffsetCurveByDistances_  
-\* _IfcSegmentedReferenceCurve_ as a 3D curve defined relative to an _IfcGradientCurve_ to incorporate the application of cant 
-\* _IfcPolyline_ as a 3D alignment by a 3D polyline representation (such as coming from a survey)  
-\* _IfcPolyline_ as a 2D horizontal alignment by a 2D polyline representation (such as in very early planning phases or as a map representation)  
+* _IfcGradientCurve_ as a 3D horizontal and vertical alignment (represented by their alignment segments)  
+* _IfcCompositeCurve_ as a 2D horizontal alignment (represented by its horizontal alignment segments) without a vertical alignment  
+* _IfcOffsetCurveByDistances_ as a 2D or 3D curve defined relative to an _IfcAlignmentCurve_ or another _IfcOffsetCurveByDistances_  
+* _IfcSegmentedReferenceCurve_ as a 3D curve defined relative to an _IfcGradientCurve_ to incorporate the application of cant 
+* _IfcPolyline_ as a 3D alignment by a 3D polyline representation (such as coming from a survey)  
+* _IfcPolyline_ as a 2D horizontal alignment by a 2D polyline representation (such as in very early planning phases or as a map representation)  
   
 > NOTE&nbsp; Although _Axis_ is an _IfcCurve_ base type, only derived types _IfcGradientCurve_, _IfcSegmentedReferenceCurve_, _IfcOffsetCurveByDistances_, and _IfcPolyline_ are meant to be supported types. Derivative specifications (Model View Definitions) may expand this set to include additional curve types.
```

Comment:
_Axis_

Reply:
This is previous text. Please provide a better description based on the final schema. @larswik @czapplitec ?